translations/en.json

```diff
@@ -253,65 +253,65 @@
   "editor.record.form.license.odc-by": "Open Data Commons ODC-By",
   "editor.record.form.license.pddl": "Open Data Commons PDDL",
   "editor.record.form.license.unknown": "Unknown or absent",
-  "editor.record.form.page.accessAndContact": "Access and contact",
-  "editor.record.form.page.description": "Resource description",
+  "editor.record.form.page.accessAndContact": "Information for use",
+  "editor.record.form.page.description": "Dataset description",
   "editor.record.form.page.ressources": "Resources",
-  "editor.record.form.section.about.description": "This section describes the resource.",
-  "editor.record.form.section.about.label": "About the resource",
-  "editor.record.form.section.annexes.description": "Annexes are optionnal. They are attached resources to the metadata record, which can help to understand the data (manual, etc...).",
+  "editor.record.form.section.about.description": "This section describes the dataset.",
+  "editor.record.form.section.about.label": "About the dataset",
+  "editor.record.form.section.annexes.description": "Annexes are optional resources. They are attached to the dataset, and help understand the data (manual, etc.).",
   "editor.record.form.section.annexes.label": "Annexes",
-  "editor.record.form.section.associatedResources.description": "Drop files here to associate them with the resource.",
+  "editor.record.form.section.associatedResources.description": "Point out where the data can be found.",
   "editor.record.form.section.associatedResources.label": "Associated resources",
-  "editor.record.form.section.classification.description": "The classification has an impact on the access to the data.",
-  "editor.record.form.section.classification.label": "Classification",
+  "editor.record.form.section.classification.description": "Appropriate keywords help users find the dataset.",
+  "editor.record.form.section.classification.label": "Keywords",
```

Comment:
This might be a little too narrow; classification can also be general themes for instance (even though we don't show it yet)